src/models/ui.ts

```diff
@@ -7,6 +7,7 @@ export const UIModel = types
     learningLogExpanded: false,
     leftNavExpanded: false,
     myWorkExpanded: false,
+    error: types.maybeNull(types.string),
```

Comment:
Is there a reason to prefer `maybeNull()` over `maybe()` here? MST seems to suggest that the latter should generally be preferred as part of a general move away from use of null.